project/build.scala

```diff
@@ -114,6 +114,7 @@ object RhoBuild extends Build {
         license,
 
         libraryDependencies <++= scalazVersion { sz => Seq(
+          shapeless,
```

Comment:
As luck would have it, parboiled2 2.1.4 appears to be using shapeless 2.3.2! **However**, they don't seem to have a 2.10 build for 2.1.4, so http4s will be out in the cold. Bummer.